docs/installation.rst

```diff
@@ -26,12 +26,14 @@ For **Debian** and **Ubuntu**, the following command will ensure that
 the required dependencies are installed: ::
 
     sudo apt-get install build-essential libssl-dev libffi-dev python-dev python-pip libsasl2-dev libldap2-dev
+    sudo apt-get install libmysqlclient-dev
 
 For **Fedora** and **RHEL-derivatives**, the following command will ensure
 that the required dependencies are installed: ::
 
     sudo yum upgrade python-setuptools
     sudo yum install gcc gcc-c++ libffi-devel python-devel python-pip python-wheel openssl-devel libsasl2-devel openldap-devel
+    sudo yum install mysql-devel
```

Comment:
```
If you are planning on using MySQL as the backend for superset, you'll need this OS-level dependency
```

Reply:
The `mysqlclient` Python package is not optional when running tests (i.e running `tox -e <py_version>`). So AFAICT this will break for everyone who wants to run the tests (which it did for me) if they don't have the `mysql_config` script.

However, this patch is not the right place for this as this `docs/installation.rst` is for the actual installation. I know little about Tox, but I can't see where there is a natural place where OS deps are handled (which is not surprising). Having a Tox plugin for handling this would be nice. Wrapping the whole thing in a script like `./run-tests -e <py_env>` where the OS deps can be handled might be an option ?